src/gui/optionsdlg.ui

```diff
@@ -1769,6 +1769,19 @@
                 </layout>
                </item>
                <item>
+                <widget class="QPushButton" name="BanListButton">
+                 <property name="sizePolicy">
+                  <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
+                   <horstretch>0</horstretch>
+                   <verstretch>0</verstretch>
+                  </sizepolicy>
+                 </property>
+                 <property name="text">
+                  <string>Manually banned list</string>
```

Comment:
The text says that you've banned a list. Perhaps,  "Manually banned IPs" would be better?. Otherwise, please provide disambiguation for translators (what the list is).